src/stores/core/selectors.ts

```
@@ -0,0 +1,92 @@
+     *
+     * @return { boolean| null } If not resolved yet then `null`.
+     */
+    hasCap( state: CoreState, capability: string ): boolean | null {
+        if (
+            ! state.currentUser.capabilities ||
+            ! state.currentUser.extra_capabilities
+        ) {
+            return null;
+        }
+
+        if ( state.currentUser.capabilities.hasOwnProperty( capability ) ) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)